DolbyIO/Source/Private/DolbyIOVideoSink.cpp

```diff
@@ -72,12 +69,13 @@ namespace DolbyIO
 
 	void FVideoSink::Convert(dolbyio::comms::video_frame& VideoFrame)
 	{
+		auto Buffer = reinterpret_cast<uint8_t*>(Texture->GetPlatformData()->Mips[0].BulkData.Lock(LOCK_READ_WRITE));
```

Comment:
Possibly `const auto` or `uint8_t* const` instead of just `auto`. Doesn't really matter though.

Reply:
In case of casts the type is obvious so `auto` is perfect (it also means `*`) - to express also `const` I can name the type as it so short.